FlightReservation/components/formComponent.js

```diff
@@ -0,0 +1,180 @@
+                <View>                    
+                <Text style={styles.inputHeader}>First Name</Text>
+                <TextInput
+                    style={[styles.inputStyle, { borderColor: borderColor.borderColorName }]}
+                    value={objValues.name}
+                    onChange={(e) =>
+                        checkInputsIsNull(e, 'name', 'borderColorName')
+                    }></TextInput>
+                </View>
+            }
+            <View style={{ flexDirection: 'row' }}>
+                <Text style={styles.inputHeader}>Email</Text>
+                <Text style={validEmail ? { color: "gray" } : styles.errorText}>*</Text>
+                {!validEmail && <Text style={styles.errorText}>Email in use. Use a different email</Text>}
+            </View>
+            <TextInput
+                style={[styles.inputStyle, { borderColor: borderColor.borderColorEmail }]}
+                value={objValues.email}
+                onChange={(e) =>
+                    checkInputsIsNull(e, 'email', 'borderColorEmail')
+                }></TextInput>
+            <View style={{ flexDirection: 'row' }}>
+                <Text style={styles.inputHeader}>Password</Text>
+                <Text style={validPassword ? { color: "gray" } : styles.errorText}>*</Text>
+                {!validPassword && <Text style={styles.errorText}>Incorrect email and/or password</Text>}
+            </View>
+            <View style={styles.containerPassword}>
+                <TextInput
+                    style={[
+                        styles.inputPassword,
+                        {
+                            borderTopColor: borderColor.borderColorPassword,
+                            borderBottomColor: borderColor.borderColorPassword,
+                            borderLeftColor: borderColor.borderColorPassword,
+                        },
+                    ]}
+                    secureTextEntry={isSeePassword}
+                    value={objValues.password}
+                    onChange={(e) =>
+                        checkInputsIsNull(e, 'password', 'borderColorPassword')
+                    }
+                />
+
+                <View
+                    style={[
+                        styles.containerIconPassword,
+                        {
+                            borderRightColor: borderColor.borderColorPassword,
+                            borderTopColor: borderColor.borderColorPassword,
+                            borderBottomColor: borderColor.borderColorPassword,
+                        },
+                    ]}>
+                    <Pressable onPress={changeSeePassword}>
+                        <Image
+                            style={[
+                                styles.iconSeePassword,
+                                { tintColor: borderColor.borderColorPassword },
+                            ]}
+                            source={require('../img/ojo-grey.png')}
```

Comment:
All the images have to be inside the import file. you only need to import the requested file by constant

Reply:
Done!